packages/components/src/ProgressBar/ProgressBar.css

```diff
@@ -1,6 +1,5 @@
 :root {
   --progress-bar--borderRadius: 1rem;
```

Comment:
We should update this to be a percentage I think so that the radius scales properly with the height.